helm/fcp-fd-data/values.yaml

```diff
@@ -58,3 +58,8 @@ usePodIdentity: true
 azureIdentity:
   resourceID: not-a-real-clientID
   clientID: not-a-real-clientID
+
+ingress:
+  class: nginx
+  endpoint: ffc-fd-data
+  server: example.com
```

Comment:
Had a previous discussion with John when I was having database issues and in general we want to avoid default values in our Helm chart. If a value is missing from appConfig, Key Vault etc. but there's a default value then the Helm chart will use the default value and deployments may look successful but we won't know for sure if the Helm chart is using the real value or this default placeholder one until we encounter an error which may not tell us the full issue. To ensure fast failure would recommend the following update:

From this...
```
server: example.com
```
to this...
```
server:
```

Can also see you have a default value for the `postgresUser` a couple lines above - this will need to be removed for the same reason e.g.:

Change this...
```
postgresUser: postgres
```
to this...
```
postgresUser:
```

Reply:
Will add that the `postgresUser` comment isn't part of this PR, but it something that should be updated.

Reply:
Also as a warning, if removing the default value for `postgresUser` causes errors during deployment because the Helm chart can't resolve the secret for the Postgres user from Key Vault then would say I'm happy to keep in the default value temporarily, then get something on the backlog to address this and ensure the relevant CCoE requests are raised to add the Postgres user for `fcp-fd-data` to the Key Vaults across all environments.

If the above has already been done, then removing the default value shouldn't cause any issues 👍🏽.

Reply:
yeah we have not done that, so will require a tech-debt ticket

Reply:
In that case leave the default value in the Helm chart for now since it's not in scope for this ticket and just ensure something is put on the backlog for raising requests for the Postgres user to be added into the `PRE` & `PRD` KVs (will need a separate request for `PRD`). Worth checking if the secret has already been added to the `DEV` & `TST` KVs though - if not you should have access to add the secret yourself.

If not done already there should also be a KV reference to the secret in appConfig that should be added via the platform repo to all environments.

Reply:
i think we've done this for PRE and PRD but it will cause failures in SND due to no value in app config.

Good spot and John mentioned similar to me so I will create a ticket for this

Reply:
Great! In that case I'm just going to go ahead and mark this as resolved.

Reply:
ticket: https://eaflood.atlassian.net/browse/SFDP-115